src/cli/mod.rs

```diff
@@ -46,7 +46,16 @@ pub struct DftCli {
     )]
     pub files: Vec<PathBuf>,
 
-    #[clap(short, long, help = "Path to the configuration file")]
```

Comment:
this does change the meaning of the `-c` argument from `-c config` to `-c <command>` to follow datafusion-cli (and psql)